collection-scripts/gather_haproxy_config

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="must-gather"
+HAPROXY_CONFIG_PATH=${OUT:-"${BASE_COLLECTION_PATH}/haproxy"}
+
+mkdir -p "${HAPROXY_CONFIG_PATH}"/
+
+function gather_haproxy_config {
+    echo "INFO: Gathering HAProxy config files"
+    INGRESS_CONTROLLERS=$(oc get ingresscontroller -n openshift-ingress-operator --no-headers -o custom-columns=":metadata.name")
+    for IC in ${INGRESS_CONTROLLERS}; do
+        POD_STRING=$(oc get pods -n openshift-ingress --no-headers -o custom-columns=":metadata.name" --selector ingresscontroller.operator.openshift.io/deployment-ingresscontroller="${IC}")
+        PODS=($POD_STRING)
+        # Use first pod per ingress controller
+        oc cp openshift-ingress/"${PODS[0]}":/var/lib/haproxy/conf/haproxy.config "${HAPROXY_CONFIG_PATH}"/"${IC}"/haproxy.config
```

Comment:
It might be useful to have `haproxy.config` from each pod in some cases.

Reply:
Right, I was going back on fourth on this. But you are correct, in the event of 1/n routers becoming wedged, the configs may be different (and having all of them would help with debugging this case).

Reply:
`7d138e0` gathers each pod's HAProxy config using a for loop.